inference/core/workflows/execution_engine/v1/dynamic_blocks/block_assembler.py

```diff
@@ -79,6 +73,16 @@ def compile_dynamic_blocks(
     return compiled_blocks
 
 
+def ensure_dynamic_blocks_allowed(dynamic_blocks_definitions: List[dict]) -> None:
+    if dynamic_blocks_definitions and not ALLOW_CUSTOM_PYTHON_EXECUTION_IN_WORKFLOWS:
+        raise WorkflowEnvironmentConfigurationError(
+            public_message="Cannot use dynamic blocks with custom Python code in this installation of `workflows`. "
+            "This can be changed by setting environmental variable "
+            "`ALLOW_CUSTOM_PYTHON_EXECUTION_IN_WORKFLOWS=True`",
+            context="workflow_compilation | dynamic_blocks_compilation",
+        )
```

Comment:
Looks like `ensure_dynamic_blocks_allowed` is only used once, and `dynamic_blocks_definitions` is guaranteed to be set at the point `ensure_dynamic_blocks_allowed` is called - maybe adding extra function call is not necessary? Is this method to be used elsewhere in the future?

Reply:
It is used twice, tests were failing without that addition, but after investigation turned out that this is only due to our way of mocking env flags. Basically introduced for convenience of testing and due to paranoid fear of RCE on the platform 😆 I would leave as is if you do not mind